### PR TITLE
Added an admin notification for new name tracking notifications with nonblank template.

### PR DESCRIPTION
**Pull request by pellaea**
This includes both new notifications with non-blank template, and updating existing notifications where the template changes from blank to non-blank.

I didn't add any new tests, because this really isn't important, and I don't expect it to stick around for long.

### CODE REVIEW

File: app/controllers/name_controller.rb [hunk 1]
```diff
@@ -1175,6 +1176,21 @@ def submit_tracking_form(name_id)
     redirect_with_query(action: "show_name", id: name_id)
   end
 
+  def notify_admins_of_notification(notification)
+    return if notification.note_template.blank?
+    return if !notification.new_record? &&
+              notification.note_template_before_last_save.blank?
```
Comment (JoeCohen) [inline app/controllers/name_controller.rb:1182]: Does the method `note_template_before_last_save` already exist?

**Comment by JoeCohen** [comment]
Looks good. Just the one question I added above.

File: app/controllers/name_controller.rb (hunk 1 above)
Comment (pellaea) [reply app/controllers/name_controller.rb:1182]: Yes, it's an automagic method.  Before you save an ActiveRecord instance, you can access the original value of any attribute "attr" by tacking on "_before_last_save", that is "instance.attr_before_last_save".  (It used to be "attr_was", but that is now deprecated.)

**Comment by JoeCohen** [comment]
Thanks! I'm glad to learn that.

**Pull request merged**